boards/arm/rp2040/raspberrypi-pico/src/rp2040_bringup.c

```diff
@@ -37,6 +37,14 @@
 #   include "rp2040_lcd_backpack.h"
 #endif
 
+#ifdef CONFIG_LCD
+#    include <nuttx/board.h>
```

Comment:
```suggestion
#include <nuttx/board.h>
```
nit: Inconsistent alignment